docs/guides/troubleshooting.md

```diff
@@ -228,3 +228,20 @@ There could be different reasons for this error:
 ### Provider "registry.terraform.io/databricks/databricks" planned an invalid value for ...: planned value ... for a non-computed attribute.
 
 Starting with version v1.51.0, the Terraform provider for Databricks supports `terraform` versions 1.1.5 and later. Older versions of `terraform`, such as v0.15.5, are known to erroneously generate this error. Check the version of `terraform` that you're using by running `terraform version` and upgrade it if necessary.
+
+### Error: cannot create ....: invalid Databricks Account configuration
+
+`....` is the descriptive name of a resource such as `access control rule set`. The error occurs when creating a workspace resource with a provider containing the `account_id` argument e.g.:
```

Comment:
instead of `access control rule set` use exact resource name, `databricks_access_control_rule_set`

Reply:
No problem updating this text.  Not sure if this is intended, but when the error surfaces with an actual `databricks_access_control_rule_set` resource, the literal text in the error contains `access control rule set` and not the resource name.  Should this be updated in a separate work stream?